citro3d/examples/triangle.rs

```diff
@@ -59,18 +59,33 @@ fn main() {
 
     let mut top_screen = gfx.top_screen.borrow_mut();
     let frame_buffer = top_screen.get_raw_framebuffer(Side::Left);
+    let (width, height) = (frame_buffer.width, frame_buffer.height);
 
     let mut instance = citro3d::Instance::new().expect("failed to initialize Citro3D");
 
-    let mut render_target = instance
-        .render_target_for_screen(
-            &frame_buffer,
-            ColorFormat::RGBA8,
-            DepthFormat::Depth24Stencil8,
-        )
-        .expect("failed to create render target");
+    let mut render_target = citro3d::render::Target::new(
+        width,
+        height,
+        &mut *top_screen,
+        DepthFormat::Depth24Stencil8,
+    )
+    .expect("failed to create render target");
+
+    render_target.set_output(Side::Left);
```

Comment:
It could, but the underlying API does allow you to set a different width/height. One use case I could think of is if you wanted to support a splitscreen camera view of a scene, or render a 2d overlay on part of the screen, something like that. 

From some brief testing, there seems to be some weird flickering / repeating effects when the render target isn't the same _width_ as the frame buffer, but it might just be me misusing the APIs, or simply a `citro3d` bug.